Source/Fuse.Common/SystemFont.uno

```diff
@@ -42,7 +42,7 @@ namespace Fuse
 			Light,
 			Normal,
 			Medium,
-			SemiBold,
+			Semibold,
```

Comment:
Here we're renaming a publicly visible entity, which is a breaking change.

That's fine if we properly deprecate the old entity by adding the following at the end of the enum:
```
[Obsolete]
SemiBold = Semibold
```

Then potential users of the old code will get a warning in their build log, instead of build errors.

Reply:
Like so right?
```
                public enum Weight
		{
			UltraLight,
			Thin,
			Light,
			Normal,
			Medium,
			Semibold,
			Bold,
			Heavy,
			Black,

			[Obsolete]
			SemiBold = Semibold
		}
```

Reply:
Yeah :)